ReplayBrowser/appsettings.json

```diff
@@ -17,23 +17,23 @@
     },
```

Comment:
Please revert the changes in appsettings.json, the current deploy script to not overwrite this, so this results in no downloads being done.

Reply:
i fucking triple checked that i didnt push that, what the fuck

Reply:
should have checked 4 times :godo: